data/src/main/java/com/globant/equattrocchio/data/ImagesServicesImpl.java

```diff
@@ -68,6 +68,32 @@ public void onFailure(Call<ImageResponse> call, Throwable t) {
         });
     }
 
+    @Override
+    public void saveImage(final Observer<List<Image>> observer) {
```

Comment:
saveImages / fetchAndSave (it's already in the ImagesServices)

Reply:
What is the difference between this method: getLatestImages

Reply:
The difference is that this method, once you get the images, stores them in a Realm database.